androbot/tests.py

```diff
@@ -7,9 +7,15 @@
 from androbot.errors import NoNewQuestionsException, UserExistsException, UserNotExistsException, WrongBotScoreFormat
 from androbot.schemas import Answer, EventsLog, Question, TelegramUser
 from androbot.types_ import AnswerTypes, Events, Specialty
+from androbot.types_.user_score import UserScore
 from androbot.utils import Utils
 
 
+@pytest.fixture()
+def act():
+    return Actions()
```

Comment:
Не совсем правильно

Надо так
```suggestion
@pytest.fixture()
def act():
    action = Actions()
    yield action
    action.__close__()
```

Дело в том, что все что определено до yield включительно, будет выполнено перед тестом, а то что после yield, независимо от успеха теста, после его выполнения